R/query.R

```diff
@@ -177,14 +177,14 @@ group_to_sce <- function(i, df, dir_prefix, features) {
     sce <- loadHDF5SummarizedExperiment(sce_path)
     # The cells we select here are those that are both available in the SCE
     # object, and requested for this particular file
-    cells <- colnames(sce) |> intersect(df$`_cell`)
+    cells <- colnames(sce) |> intersect(df$`cell_`)
```

Comment:
Remove the backticks from here and below to prove that it is a valid identifier

Reply:
Done